packages/user/Sites/include/services/user/Sites.hpp

```diff
@@ -44,11 +44,11 @@ namespace SystemService
    struct SiteConfigRow
    {
       psibase::AccountNumber     account;
-      bool                       spa       = false;
-      bool                       cache     = true;
-      std::optional<std::string> globalCsp = std::nullopt;
+      bool                       spa;
+      bool                       doNotCache;
```

Comment:
> invert site config cache logic so default value can be false

The tail is wagging the dog here. Negative boolean variables tend to be more confusing to read because they result in double negatives.

Reply:
Reverted